src/server/scripts/Shadowlands/Sanctum of Domination/boss_sylvanas_windrunner.cpp

```diff
@@ -0,0 +1,3694 @@
+{
+    VISUAL_KIT_SYLVANAS_INTRODUCTION_TALK_01      = 150067,
+    VISUAL_KIT_SYLVANAS_INTRODUCTION_TALK_02      = 150068,
+    VISUAL_KIT_SYLVANAS_INTRODUCTION_TALK_03      = 150069,
+    VISUAL_KIT_SYLVANAS_INTRODUCTION_TALK_04      = 150071,
+    VISUAL_KIT_SYLVANAS_INTRODUCTION_TALK_05      = 150072,
+    VISUAL_KIT_SYLVANAS_INTRODUCTION_TALK_06      = 150070,
+    VISUAL_KIT_SYLVANAS_INTRODUCTION_TALK_07      = 150074,
+    VISUAL_KIT_SYLVANAS_INTRODUCTION_TALK_08      = 150077,
+    VISUAL_KIT_SYLVANAS_INTRODUCTION_TELEPORT     = 150078,
+    VISUAL_KIT_SYLVANAS_INTRODUCTION_TALK_09      = 150076,
+    VISUAL_KIT_SYLVANAS_INTRODUCTION_TALK_10      = 150075,
+
+    VISUAL_KIT_SYLVANAS_NORMAL_CAST               = 142405,
+    VISUAL_KIT_SYLVANAS_UNSHEATHE_BOW             = 143939,
+    VISUAL_KIT_SYLVANAS_UNSHEATHE_BOW_SPIN        = 142389,
+    VISUAL_KIT_SYLVANAS_UNSHEATHE_DAGGERS         = 143940,
+    VISUAL_KIT_SYLVANAS_UNSHEATHE_DAGGERS_SPIN    = 142388,
+    VISUAL_KIT_SYLVANAS_WAILING_ARROW_CHARGE      = 142447,
+    VISUAL_KIT_SYLVANAS_WAILING_ARROW_JUMP        = 145145,
+    VISUAL_KIT_SYLVANAS_WAILING_ARROW_EFFECT      = 146199,
+    VISUAL_KIT_SYLVANAS_VEIL_OF_DARKNESS          = 142021,
+    VISUAL_KIT_SYLVANAS_RIVE_BREAK                = 145541,
+    VISUAL_KIT_SYLVANAS_RIVE_BREAK_FAST           = 145608,
+    VISUAL_KIT_SYLVANAS_TRANSFORM_INTO_BANSHEE    = 146063,
+    VISUAL_KIT_SYLVANAS_BANSHEE_SCREAM_EXPIRE     = 148298,
+    VISUAL_KIT_SYLVANAS_UNK_17                    = 149160, // Does nothing
+    VISUAL_KIT_SYLVANAS_RUIN_INTERRUPTED          = 145486,
+    VISUAL_KIT_SYLVANAS_BANSHEE_WAVE              = 148270,
+    VISUAL_KIT_SYLVANAS_TRANSFORM_INTO_HUMAN      = 146626,
+    VISUAL_KIT_SYLVANAS_SHADOW_DAGGER             = 145482,
+    VISUAL_KIT_SYLVANAS_SHADOW_DAGGER_SOUND       = 145850,
+    VISUAL_KIT_SYLVANAS_SHOOT                     = 149795, // Withering Fire on phase 2?
+    VISUAL_KIT_SYLVANAS_BANSHEE_VEIL_OF_DARKNESS1 = 144927,
+    VISUAL_KIT_SYLVANAS_UNK_18                    = 149161, // Does nothing
+    VISUAL_KIT_SYLVANAS_DARK_FOG                  = 150066,
+    VISUAL_KIT_SYLVANAS_STOMP                     = 148200,
+    VISUAL_KIT_SYLVANAS_SWITCH_PLATFORM           = 149580,
+    VISUAL_KIT_SYLVANAS_BANSHEE_SCREAM            = 146059,
+    VISUAL_KIT_SYLVANAS_BANSHEE_BANE              = 145872,
+    VISUAL_KIT_SYLVANAS_BLACKEN_WITH_DAGGERS      = 149487, // NOT SNIFFED
+    VISUAL_KIT_SYLVANAS_DISAPPEAR_MODEL           = 149491, // NOT SNIFFED
+    VISUAL_KIT_SYLVANAS_VEIL_OF_DARKNESS_3_01     = 148191,
+    VISUAL_KIT_SYLVANAS_VEIL_OF_DARKNESS_3_02     = 148193,
+    VISUAL_KIT_SYLVANAS_VEIL_OF_DARKNESS_3_03     = 148192,
+
+    VISUAL_KIT_SYLVANAS_BANSHEE_VEIL_OF_DARKNESS2 = 144927, // Same as 17
+
+    VISUAL_KIT_JAINA_KNEEL_THEN_STAND             = 146285,
+    VISUAL_KIT_JAINA_CHEERS                       = 147631,
+    VISUAL_KIT_JAINA_FROZEN_BRIDGE                = 142472, // NOT ON SNIFF
```

Comment:
I found it out at some point during some sorting of the visuals and I thought I might need it at some point later during phase 2 development. It's just there in case, might just delete it if in the end, I don't need it, but since I've been dealing with so many spells with missing visuals, this could also be the case.